homeassistant/components/overkiz/cover_entities/awning.py

```diff
@@ -7,11 +7,11 @@
 
 from homeassistant.components.cover import (
```

Comment:
> If the state is static why do we need it?
> 
> If it's a binary state it should probably be a binary sensor instead.
> 
> _Originally posted by @MartinHjelmare in https://github.com/home-assistant/core/pull/64564#discussion_r793235961_

Good one. I will have a better look at this value and why we don't expose it as a binary_sensor / sensor yet! However, this code should be there as well since HomeKit still needs this attribute to set 'obstruction detected' on a specific cover. 

Will double check the state value, but I recall that this state doesn't exist (only in definition) until the cover is blocked.